src/pages/workspaces/workspace/Data.js

```diff
@@ -885,7 +883,11 @@ const WorkspaceData = _.flow(
                       })
                     })
                   ])
-                }, wdsTypes.state)
+                }, wdsTypes.state),
+                h(NoDataPlaceholder, {
+                  buttonText: 'Data Table Status',
+                  onAdd: () => setTroubleshootingWds(true)
```

Comment:
Wondering if we should change the naming to setCheckingWdsStatus, and change from WdsTroubleshooter to WdsStatusModal. My vote is to keep things as they are-